extensions/clusters/kubernetesversions/all.go

```diff
@@ -222,9 +202,8 @@ func ListEKSAllVersions(client *rancher.Client) (allAvailableVersions []string,
 
 // sortReleases is a private function that sorts release structs that are used for K3S and RKE2.
 // Sorted versions determined by these conditions:
-//  1. Current rancher version is between min and max channel versions
-//  2. Release struct has serverArgs and agentArgs not empty fields
-//  3. Possible newest version of the minimum channel version
+//  1. Release struct has serverArgs and agentArgs not empty fields
+//  2. Possible newest version of the minimum channel version
 func sortReleases(rancherVersion *semver.Version, releases []interface{}) (allAvailableVersions []string) {
```

Comment:
The function arguments are updated in the other ports, could you rebase this PR?

Reply:
Thanks for catching that! It should be fixed now.